ClimateMonitorV2/public/stylesheets/main.css

```diff
@@ -28,14 +28,16 @@ body {
 
 /* Sidebar setup */
 .sidebar {
```

Comment:
Could we make the sidebar page names bold? I find it a little hard to read in the current format (Using [font-weight](https://www.w3schools.com/cssref/pr_font_weight.asp) will probably work well)